emails/migrations/0050_profile_store_phone_log.py

```diff
@@ -0,0 +1,127 @@
+# Generated by Django 3.2.14 on 2022-08-05 14:28
+
+from django.db import migrations, models
+
+
+def add_db_default_forward_func(apps, schema_editor):
+    """
+    Add a database default of TRUE for store_phone_log
+
+    Using `./manage.py sqlmigrate` for the SQL, and the technique from:
+    https://stackoverflow.com/a/45232678/10612
+    """
+    if schema_editor.connection.vendor.startswith("postgres"):
+        schema_editor.execute(
+            'ALTER TABLE "emails_profile"'
+            ' ALTER COLUMN "store_phone_log" SET DEFAULT true;'
+        )
+    elif schema_editor.connection.vendor.startswith("sqlite"):
+        schema_editor.execute(
+            'CREATE TABLE "new__emails_profile"'
+            ' ("id" integer NOT NULL PRIMARY KEY AUTOINCREMENT,'
+            ' "api_token" char(32) NOT NULL,'
+            ' "user_id" integer NOT NULL REFERENCES "auth_user" ("id") DEFERRABLE INITIALLY DEFERRED,'
+            ' "address_last_deleted" datetime NULL,'
+            ' "num_address_deleted" integer unsigned NOT NULL CHECK ("num_address_deleted" >= 0),'
+            ' "last_hard_bounce" datetime NULL,'
+            ' "last_soft_bounce" datetime NULL,'
+            ' "subdomain" varchar(63) NULL UNIQUE,'
+            ' "server_storage" bool NOT NULL,'
+            ' "store_phone_log" bool NOT NULL,'
```

Comment:
Nice job with this messiness! One change, add the database-level default of `true`:

```suggestion
            ' "store_phone_log" bool NOT NULL DEFAULT 1,'
```